microsoft-edge/devtools-guide-chromium/whats-new/2023/08/devtools-116.md

```diff
@@ -0,0 +1,115 @@
+---
+title: What's new in DevTools (Microsoft Edge 116)
+description: "Move the location of the Activity Bar in Focus Mode. React Developer Tools extensions works in Focus Mode.  Containment Diff view in the Memory tool when multiple heap snapshots.  And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/01/2023
+---
+# What's New in DevTools (Microsoft Edge 116)
+
+<!-- todo: ms.date above: put date of 116 Stable -->
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## "Move Activity Bar" button in the Activity bar
+
+<!-- Subtitle: __ -->
```

Comment:
add Subtitle sentence
@leahmsft

Reply:
won't do since we won't add to welcome tool